### ISSUE TITLE
onHangupCompleted is never called after hangup

### ISSUE THREAD

**Issue by stevkan** [microsoft/BotBuilder-Samples#277]
### SDK Platform
.NET

### URL of Code Sample
[https://github.com/Microsoft/BotBuilder-Samples/tree/master/CSharp/skype-CallingBot](https://github.com/Microsoft/BotBuilder-Samples/tree/master/CSharp/skype-CallingBot)

### Problem Description
Issue [#4743](https://github.com/Microsoft/BotBuilder/issues/4743) originally posted on Microsoft/Botbuilder repository.

onHangupCompleted in IVRBot.cs file is never called when hangup occurs by user or bot. 

```cs
private Task OnHangupCompleted(HangupOutcomeEvent hangupOutcomeEvent)
{
    System.Diagnostics.Trace.TraceInformation(hangupOutcomeEvent.ConversationResult.Id);
    hangupOutcomeEvent.ResultingWorkflow = null;

    return Task.FromResult(true);
}
```

onHangupCompleted calls the callback url:
Requested URL | https://xxxxxxxxxxx:80/api/calling/callback
Other callbacks point to:
Requested URL | https://xxxxxxxxxxx.azurewebsites.net:80/api/calling/callback

**Comment by stevkan** [comment]
Working on recreating the bug. Presently, the sample does not work, as-is. Have located certain breaks in the code and implementing fixes. Next, removing Bing Speech as the text-to-speech service as it is deprecated and adding Microsoft CognitiveServices Speech.

**Comment by CoHealer** [comment]
@stevkan, what is the current status on this?

**Comment by stevkan** [comment]
Current code won't progress beyond the initial bot sendActivityAsync. I suspect it is tied to the call api and how the response is being handled. Am trying to narrow down.

**Comment by stevkan** [comment]
Updated the calling REST endpoint and adjusted the callback url. Documentation states that, with the move from Bing Speech to Cognitive Services Speech, all apis and methods are remaining the same. Only the calling endpoint is different. Bot now moves beyond the initial message. However, some aspect of Skype appears to have changed as the bot will not respond to any dial pad input from the user.

The recognizer should detect DTMF tones when a number is input. The tones, via Skype, don't produce what would be considered a typical sound. However, when calling my phone, the output received by the phone produces a typical button sound. A simple search on the internet shows various related issues around Skype and DTMF over the last few years but no clear resolution. That being said, this calling bot worked around June of last year with no issue, though the UX was different (in appearance, at least) than it is now.

I have tried placing breakpoints across various methods but they are never called as input is never recognized. As such, I cannot get to a point where I can test the actual onHangupCompleted issue.
